pkgs/by-name/pl/plasma-panel-colorizer/package.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   buildInputs = [ kdePackages.plasma-desktop ];
-
+  propagatedUserEnvPkgs = [ glib ];
```

Comment:
I see the only other instance of this pattern in nixpkgs is also authored by you. Could you explain why you opt to use this instead of suffixing the PATH with a wrapper?

Reply:
:+1: